frontend/app/src/components/home/upgrade/Payment.svelte

```diff
@@ -101,6 +123,25 @@
             })
             .finally(() => (confirming = false));
     }
+
+    onMount(() => {
+        diamondFees = { kind: "loading" };
+        client
+            .diamondMembershipFees()
+            .then((fees) => {
+                diamondFees = {
+                    kind: "success",
+                    data: client.toRecord2(
```

Comment:
I'm pretty sure this is what `toRecord` does, as in it sets the input as the value.